src/main/java/com/checkmarx/flow/CxFlowRunner.java

```diff
@@ -1,10 +1,7 @@
 package com.checkmarx.flow;
 
 import com.checkmarx.flow.config.*;
-import com.checkmarx.flow.dto.BugTracker;
-import com.checkmarx.flow.dto.ExitCode;
-import com.checkmarx.flow.dto.FlowOverride;
-import com.checkmarx.flow.dto.ScanRequest;
+import com.checkmarx.flow.dto.*;
```

Comment:
please avoid using * in imports

Reply:
IntelliJ has a functionality that allows to quickly optimize imports with a single hotkey. It produces * when it considers this relevant. Making manual changes in the import statements will take much time and will be overwritten by the next invocation of 'optimize imports' anyway.